cpp/src/arrow/c/bridge.cc

```diff
@@ -444,6 +444,10 @@ struct SchemaExporter {
 
   Status Visit(const LargeListType& type) { return SetFormat("+L"); }
 
+  Status Visit(const ListViewType& type) { return SetFormat("+lv"); }
+
+  Status Visit(const LargeListViewType& type) { return SetFormat("+Lv"); }
```

Comment:
Looks like the vote has established
```suggestion
  Status Visit(const ListViewType& type) { return SetFormat("+vl"); }

  Status Visit(const LargeListViewType& type) { return SetFormat("+vL"); }
```

Reply:
Fixing up the commit changing the bridge code.